cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -0,0 +1,157 @@
+import {Organization} from '../../../src/types'
+
+describe('FluxQueryBuilder', () => {
+  beforeEach(() => {
+    cy.flush().then(() => {
+      cy.signin().then(() => {
+        cy.get('@org').then(({id}: Organization) => {
+          cy.visit(`/orgs/${id}/data-explorer`)
+          cy.getByTestID('tree-nav').should('be.visible')
+          cy.setFeatureFlags({
+            newDataExplorer: true,
+          }).then(() => {
+            cy.wait(100)
+            // Switch to Flux Query Builder
+            cy.getByTestID('flux-query-builder-toggle')
+              .should('be.visible')
+              .click()
+          })
```

Comment:
Seems like setting the feature flag after landed on the data explorer page give the toggle to show up